### ISSUE TITLE
 I can't create or drag items to the character sheet

### ISSUE THREAD

**Issue by PilotodeMouse** [HavlockV/CoC7-FoundryVTT#1459]
### Prerequisites.

- [X] Have you checked the Upcoming Release section of the changelog to see if the issue will be resolved in the next release? https://github.com/Miskatonic-Investigative-Society/CoC7-FoundryVTT/blob/develop/.github/CHANGELOG.md
- [X] Are you running the latest version of both Foundry VTT and Call of Cthulhu 7th Edition?
- [X] Have you tested whether the bug persists without any other modules activated or on Foundry Launch in Safe Configuration?

## Description.

When trying to create or drag an item to the form, the message appears: Unable to read properties of undefined (reading 'length'). Foundry Version 11 Stable / 315 - Call of Cthulhu 7th Edition 0.10.80

## Screenshots.

![image](https://github.com/Miskatonic-Investigative-Society/CoC7-FoundryVTT/assets/76591455/99f5fe9c-baf7-4c12-a595-7f287b49b624)

**Comment by snap01** [comment]
Can you please provide the step by step instructions to replicate this issue?

Can you also please provide the full error message in the JavaScript console (F12)

Creating a new world and importing Examples/1920 Character and then dragging skills, weapons, manias and phobias from the compendiums all seem to work without error.

**Comment by PilotodeMouse** [comment]
Thanks very much for the answer! @snap01  :)

I carried out the suggested tests and unfortunately I was unsuccessful.

I used the system without any active modules in a new world.

Below is the Error Console. I also recorded a step-by-step video of the problem.

![image](https://github.com/Miskatonic-Investigative-Society/CoC7-FoundryVTT/assets/76591455/adf7b219-73d3-430c-bec0-1aa0ab8e28ce)

[Foundry.webm](https://github.com/Miskatonic-Investigative-Society/CoC7-FoundryVTT/assets/76591455/012a8689-8f04-4e75-a63a-00bcecaa17d2)
